lib/modules/datasource/nuget/index.ts

```diff
@@ -19,15 +19,11 @@ export class NugetDatasource extends Datasource {
   override readonly registryStrategy = 'merge';
 
   override readonly releaseTimestampSupport = true;
-  override readonly releaseTimestampNote = `
-  For the v2 API, the release timestamp is determined from the \`Publised\` tag and,
-  for the v3 API, the release timestamp is determined from the \`published\` field in the results.
-   `;
+  override readonly releaseTimestampNote =
+    'For the v2 API, the release timestamp is determined from the `Publised` tag. And, for the v3 API, the release timestamp is determined from the `published` field in the results.';
```

Comment:
```suggestion
    'For the v2 API, the release timestamp is determined from the `Published` tag. And, for the v3 API, the release timestamp is determined from the `published` field in the results.';
```

I think this was a typo?